### ISSUE TITLE
removing one IC from one of clusters

### ISSUE THREAD

**Issue by tingyu653** [eeglabdevelopers/eeglab#355]
### Description
After clustering, I removed one of IC from one cluster based on ERPs. After that, I tried to see the result after removing. But I got several errors from Matlab.
![image](https://user-images.githubusercontent.com/85797007/126071581-b75b3c41-9fc5-482d-87ef-94c595858313.png)
The result also doesn't seem right. I found out that the IC I removed still exist, and the name of the IC changed into s03A/IC15.
![image](https://user-images.githubusercontent.com/85797007/126071680-9daf838c-8776-4bf7-b84b-eb1d37fe9abe.png)
This is the original ERPs before removing the first IC(s01A/IC10).
![image](https://user-images.githubusercontent.com/85797007/126072499-c2d0c618-981f-4ff4-9352-1c8fe66d176c.png)

<br>
#### Versions

|   |   |
|----| ---- | 
| OS version     | [*fill in*] |
| Matlab version | [*R2020a*] |
| EEGLAB version | [*v2020.0*] |

**Comment by cll008** [comment]
Could you please elaborate on how you removed the IC, and what process you are using to 'see the result'? Thanks.

**Comment by arnodelorme** [comment]
Would you mind sharing your STUDY, and providing a detailed description on how to reproduce the problem?

**Comment by tingyu653** [comment]
I pressed the "Reassign selected component(s) " button, and removed the IC from Cls 7 to outlier. To 'see the result', I pressed the 'Plot ERP(s)' button at right column.(Cls 7->all components) immediately.
![image](https://user-images.githubusercontent.com/85797007/126245786-a0f539d9-4cd5-46b4-9743-922670788858.png)
After I posted the question, I tried not to press the 'Plot ERP(s)' button immediately, and I saw the correct ERP(s). 
But I still wonder why couldn't I press the 'Plot ERP(s)' button immediately to see the ERP(s) after removing the IC?
